orchestra/static/orchestra/todos/todo-list.directive.es6.js

```diff
@@ -93,9 +105,10 @@ export default function todoList (orchestraApi) {
       }
 
       todoList.removeTodo = (todo) => {
-        var index = todoList.todos.indexOf(todo)
-        todoList.todos.splice(index, 1)
-        todoApi.delete(todo)
+        const idsToUpdate = getNestedTodoIds(todo)
+        remove(todoList.todos, t => idsToUpdate.includes(t.id))
```

Comment:
if `N = todoList.todos.length` and `M = idsToUpdate.length`, this is an O(N*M) search, whereas if `idsToUpdate` was a set instead of a list, it would be O(N)

Reply:
+1